Sources/docc/DocCDocumentation.docc/Reference Syntax/API Reference Syntax/api-reference-syntax.md

```diff
@@ -14,6 +14,7 @@ Use documentation markup, a custom variant of Markdown, to add in-source documen
 
 - ``Options``
 - ``Metadata``
+- ``TechnologyRoot``
```

Comment:
Is this an exhaustive list of page configuration directives? It would be nice to have a unit test of some kind double check this list, and the following list of layout directives, against the DocC source code. Then if we add a new directive someday the test will fail, reminding us to add a link to one of these lists here.

What do you think?

Reply:
This is not an exhaustive list; I added 'TechnologyRoot' to the curation to give it more visibility, as it is a key directive for 'Conceptual Documentation.'
Currently, to learn about that directive, you need to navigate through:
'API Documentation' -> 'Metadata' -> 'Technology Root'
If we are making changes to include 'Conceptual docs' as one of the possible doc types available in DocC, we should probably move the curation up one level.